curvefs/src/mds/fs.h

```diff
@@ -0,0 +1,148 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * @Project: curve
+ * @Date: 2021-06-04 14:11:09
+ * @Author: chenwei
+ */
+
+#ifndef CURVEFS_SRC_MDS_FS_H_
+#define CURVEFS_SRC_MDS_FS_H_
+
+#include <list>
+#include <string>
+#include "curvefs/proto/mds.pb.h"
+#include "src/common/concurrent/rw_lock.h"
+
+using curve::common::ReadLockGuard;
+using curve::common::WriteLockGuard;
+using curve::common::RWLock;
+using curvefs::common::Volume;
+using curvefs::common::FSType;
+using curvefs::common::S3Info;
+
+namespace curvefs {
+namespace mds {
+class MdsFsInfo {
+ public:
+    MdsFsInfo() {}
```

Comment:
Default constructor should init members which is POD variable such as  fsId_、rootInodeId_ etc.

Reply:
done